packages/gatsby-cli/README.md

```diff
@@ -16,44 +16,44 @@ package installed locally.
 
 Run `gatsby --help` for full help.
 
-### New
+### `new`
 
 `gatsby new gatsby-site`
 
 See the [Gatsby starters docs](https://www.gatsbyjs.org/docs/gatsby-starters/)
 for more.
 
-### Develop
+### `develop`
 
 At the root of a Gatsby site run `gatsby develop` to start the Gatsby
 development server.
 
 #### Options
 
-|     Option      | Description                                     |
-| :-------------: | ----------------------------------------------- |
-| `-H`, `--host`  | Set host. Defaults to localhost                 |
-| `-p`, `--port`  | Set port. Defaults to 8000                      |
-| `-o`, `--open`  | Open the site in your (default) browser for you |
-| `-S`, `--https` | Use HTTPS                                       |
+|     Option      | Description                                     |   Default   |
+| :-------------: | ----------------------------------------------- | :---------: |
+| `-H`, `--host`  | Set host.                                       | `localhost` |
+| `-p`, `--port`  | Set port.                                       |   `8000`    |
```

Comment:
not sure but you probably want this to be aligned with `localhost`

Reply:
What did you mean here? This is from prettier, don't have a ton of control over alignment here!

Reply:
seems like it got fixed, it seems like localhost was not aligned properly but now it is